.github/workflows/docker-image.yaml

```diff
@@ -73,9 +77,17 @@ jobs:
             RELEASE=${{ inputs.release }}
             COMMIT=${{ inputs.commit }}
             BUILD_DATE=${{ inputs.build_date }}
-      
+
+      - name: Install Cosign
+        if: inputs.sign
+        uses: sigstore/cosign-installer@03d0fecf172873164a163bbc64bed0f3bf114ed7 # v3.4.0
+        with:
+          cosign-release: 'v2.0.2'
+
       - name: Sign the images with GitHub OIDC Token
+        if: inputs.sign
```

Comment:
```suggestion
        if: ${{ inputs.sign }}
```

Reply:
Thanks! Honest question: what is the difference?